fastlane/Fastfile

```diff
@@ -400,7 +401,8 @@ end
         export_team_id: ENV["EXT_EXPORT_TEAM_ID"],
         provisioningProfiles: {
           "com.codality.NotationalFlow" => "match AppStore com.codality.NotationalFlow",
-          "com.codality.NotationalFlow.Share" => "match AppStore com.codality.NotationalFlow.Share"
+          "com.codality.NotationalFlow.Share" => "match AppStore com.codality.NotationalFlow.Share",
+          "#{APP_STORE_BUNDLE_IDENTIFIER}.Intents" => "match InHouse #{APP_STORE_BUNDLE_IDENTIFIER}.Intents"
```

Comment:
⚠️  I think this should be `match AppStore …` instead of `match InHouse`

Reply:
Also, is this normal that we have `#{root_bundle_id}.Widgets"` listed in `simplenote_app_identifiers` but that we dont have them listed as participating in provisioningProfiles mapping here? Aren't those missing?

Reply:
> ⚠️  I think this should be `match AppStore …` instead of `match InHouse`

ZOMG 😱 Thanks @AliSoftware great catch!

Reply:
> ⚠️  Also, is this normal that we have `#{root_bundle_id}.Widgets"` listed in `simplenote_app_identifiers` but that we dont have them listed as participating in provisioningProfiles mapping here? Aren't those missing?

I asked myself the same question when touching this code, but then I figured that "it works in production (right?)" so maybe we don't need them?

It might be interesting to add them and see what happens to the code signing and ASC submission, but that might be a change for a different PR.